README.md

```diff
@@ -75,38 +77,6 @@ While some Kubernetes distros package everything and the kitchen sink in, k0s tr
 
 With strong enough arguments we might take in new addons but in general those should be something that are essential for the "core" of k0s.
 
-## Build
-
-`k0s` can be built in 3 different ways:
-
-Fetch official binaries (except `kine` and `konnectivity-server`, which are built from source):
-```
-make EMBEDDED_BINS_BUILDMODE=fetch
-```
-
-Build Kubernetes components from source as static binaries (requires docker):
-```
-make EMBEDDED_BINS_BUILDMODE=docker
-```
-
-Build k0s without any embedded binaries (requires that Kubernetes
-binaries are pre-installed on the runtime system):
-```
-make EMBEDDED_BINS_BUILDMODE=none
-```
-
-Builds can be done in parallel:
-```
-make -j$(nproc)
-```
-
-## Smoke test
-
-To run a smoke test after build:
-```
-make check-basic
-```
-
 ## Cluster bootstrapping
 
 Move the built `k0s` binary to each of the nodes.
```

Comment:
I'd suggest adding this too:
```
curl -SsLf get.k0s.sh 
```

Reply:
I can't comment on the line below but I'd also give an example with `enable-worker`

Reply:
Do we need `--enable-worker` example in the readme when we have it in the [FAQ](https://github.com/k0sproject/k0s/blob/main/docs/FAQ.md#how-do-i-run-a-single-node-cluster)?

Reply:
I'm thinking it is most likely how someone is going to try k0s